NineChronicles.Headless/GraphTypes/StandaloneQuery.cs

```diff
@@ -72,6 +75,94 @@ public StandaloneQuery(StandaloneContext standaloneContext, IConfiguration confi
                 }
             );
 
+            Field<ListGraphType<RootStateDiffType>>(
+                name: "diffs",
+                arguments: new QueryArguments(
+                    new QueryArgument<NonNullGraphType<LongGraphType>>
+                    {
+                        Name = "baseIndex",
+                        Description = "The index of the base block used to fetch state from the chain. " +
+                                    "It should be less than or equal to the chain tip index, " +
+                                    "greater than or equal to the genesis block index (0), " +
+                                    "and less than changedIndex. The difference between baseIndex " +
+                                    "and changedIndex should not be greater than 10 and should not be zero."
+                    },
+                    new QueryArgument<NonNullGraphType<LongGraphType>>
+                    {
+                        Name = "changedIndex",
+                        Description = "The index of the target block used to fetch state from the chain. " +
+                                    "It should be less than or equal to the chain tip index, " +
+                                    "greater than or equal to the genesis block index (0), " +
+                                    "and greater than baseIndex. The difference between baseIndex " +
+                                    "and changedIndex should not be greater than 10 and should not be zero."
+                    }
+                ),
```

Comment:
The argument descriptions need to fix.
Could you add description for "diffs" query?

Reply:
yes roger that